lib/Doctrine/DBAL/Platforms/OraclePlatform.php

```diff
@@ -411,43 +411,53 @@ protected function _getCreateTableSQL($table, array $columns, array $options = a
     }
 
     /**
+     * Returns the SQL for a list of indexes in the database.
+     *
+     * @param string $database
+     * @param string $table
```

Comment:
It would be nice to have some explanation of the meaning of the parameters given they are nullable.